config/users.php

```diff
@@ -74,7 +74,7 @@
     */
 
     'passwords' => [
-        'resets' => config('auth.defaults.passwords'),
+        'users' => config('auth.defaults.passwords'),
```

Comment:
@jasonvarga Would it be considered a breaking change now that Statamic 3 is out if we changed the key of a configuration option?